lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -365,7 +365,7 @@ def test_http_exception(
             pytest.skip("skipping " + self.test_http_exception.__name__)
 
         if self.use_index_from_metadata:
-            index = self.build_index_from_metadata()
+            index = self.build_index_from_metadata("result-data-sample")
```

Comment:
Having repeated literal strings makes it easy for someone to miss a typo; using symbols is better, as the language will detect typos for you. For example, a `Commons.RESULT_DATA_SAMPLE` ... or maybe mapping our root index names as an ENUM isn't a bad idea.